src/main/java/com/browserstack/automate/ci/jenkins/pipeline/BrowserStackPipelineStepExecution.java

```diff
@@ -97,7 +97,7 @@ protected Void run() throws Exception {
 
         tracker.pluginInitialized(overrides.get(Constants.JENKINS_BUILD_TAG),
                 (this.localConfig != null), true);
-        return null;
+        return false;
```

Comment:
return `true`?

Reply:
https://javadoc.jenkins.io/plugin/workflow-step-api/org/jenkinsci/plugins/workflow/steps/StepExecution.html#start--

true if the execution of this step has synchronously completed before this method returns.
false if the asynchronous execution has started and that StepContext will be notified when the result comes in.

Reply:
Can be resolved.